lib/types.js

```diff
@@ -170,12 +199,60 @@ export type AllChatOptions =
   | ChatOptionsDownload
   | ChatOptionsAttachment
 
-/**
+/*
+ * Chat API Function Signatures
  */
-export type ChatReadMessage = any
-/**
+
+export type Init = InitOptions => Promise<void>
+
+export type Deinit = () => Promise<void>
+
+export type MyInfo = () => ?DeviceUsernamePair
+
+export type List = ChatOptionsList => Promise<void | ChatConversation[]>
+
+export type Read = ChatOptionsRead => Promise<any>
+
+export type Send = ChatOptionsSend => Promise<void>
+
+export type Delete = ChatOptionsDelete => Promise<void>
```

Comment:
How do these types relate to the function type signatures in `chat/index.js`? `deleteMessage` has a return type of `Promise<any>` over there. These look to be the more authoritative ones since they're used in the "with store" functions, but does that mean we have to duplicate function signatures?